Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/ColliderCapsuleManipulators.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/TickBus.h>
+#include <AzToolsFramework/ComponentModes/CapsuleViewportEdit.h>
+#include <Editor/Plugins/Ragdoll/PhysicsSetupManipulatorCommandCallback.h>
+#include <Editor/Plugins/Ragdoll/PhysicsSetupManipulators.h>
+#include <MCore/Source/Command.h>
+#include <MCore/Source/MCoreCommandManager.h>
+
+namespace EMotionFX
+{
+    //! Provides functionality for interactively editing character physics capsule collider dimensions in the Animation Editor Viewport.
+    class ColliderCapsuleManipulators
+        : public PhysicsSetupManipulatorsBase
+        , private AZ::TickBus::Handler
+        , private PhysicsSetupManipulatorRequestBus::Handler
+        , private AzToolsFramework::CapsuleViewportEdit
+    {
+    public:
+        ColliderCapsuleManipulators();
+        ~ColliderCapsuleManipulators();
+        void Setup(PhysicsSetupManipulatorData& physicsSetupManipulatorData) override;
+        void Refresh() override;
+        void Teardown() override;
+        void ResetValues() override;
```

Comment:
nit: it'd be nice if these override functions would have a `// PARENT_CLASS overrides ...` comment like the others.

Reply:
Added.